OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitWebViewFragment.java

```diff
@@ -132,6 +152,64 @@ public void onPageLoadProgressChanged(WebView webView, int progress) {
         }
     }
 
+    private void evaluateJavascriptForiFrame() {
+        if (!TextUtils.isEmpty(unit.getBlockId()) && !evaluatediFrameJS) {
+            // execute js code to check an HTML block that contains an iframe
+            String javascript =
+                    "try {" +
```

Comment:
any reference to this code?

Reply:
Shared by Colin Ref: https://openedx.atlassian.net/browse/LEARNER-8362?focusedCommentId=542436